stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/channel/ChannelController.kt

```diff
@@ -1053,10 +1053,8 @@ public class ChannelController internal constructor(
             ?: domainImpl.repos.selectMessage(messageId)
             ?: return Result(ChatError("Error while fetching message from backend. Message id: $messageId"))
         upsertMessage(message)
-        domainImpl.scope.launch {
```

Comment:
If this doesn't happen synchronously, the messages are not loaded before scrollTo is called, then there's no message to scroll to... then nothing happens.